testsuite/single_node_performance.py

```diff
@@ -163,6 +163,11 @@ class RunGroupConfig:
 
 HIDE_OUTPUT = os.environ.get("HIDE_OUTPUT")
 
+DATA_DIR = os.environ.get("DATA_DIR")
+if not(DATA_DIR):
+    print("Please set env_var DATA_DIR")
```

Comment:
do we want to default one, or require it to be set?

Reply:
For the purpose of pn2 I think it's better to require people to set.. just one less thing to ask each case that doesn't perform well.

Reply:
actually, this will break regression test on PRs, you definitely need to have default :)

Reply:
I mean I don't land it on main

Reply:
(if we do we update the CI workflows to put in /tmp)